.github/workflows/assets-sync.yml

```diff
@@ -14,55 +14,41 @@ jobs:
         with:
           repos: |
             pion/datachannel
+            pion/dcnet
             pion/dtls
             pion/ice
             pion/interceptor
             pion/logging
             pion/mdns
+            pion/mediadevices
```

Comment:
Mediadevices has large difference from other pion repos especially the usage of CGO, unsafe, and other raw system access.
I don't think it's reasonable to apply the shared lint rules to mediadevices.

Reply:
Agreed. I removed it.